app/javascript/dashboard/App.vue

```diff
@@ -112,9 +112,11 @@ export default {
       ) {
         this.theme = 'dark';
         document.body.classList.add('dark');
+        document.documentElement.setAttribute('style', 'color-scheme: dark;');
```

Comment:
@iamsivin Can you please share why this is needed.

Reply:
@pranavrajs With this, the browser can apply its default dark or light styling to certain elements, such as scroll bars, check boxes, and text areas.

Reply:
For example, here are some screenshots,

1. **Audio player**
<img width="353" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/86ad5a4e-a41d-4010-b953-b6b2f86fd05f">

2. **Scrollbar**

| **Darkmode**   | **Lightmode**   |
|------------|------------|
| <img width="353" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/40d285a8-2c3f-4508-888f-71a0cc1eb2b3">   | <img width="353" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/d3fc155a-8f36-457b-bdd0-c1a0edd0da29">   | 

3. **Checkbox**
<img width="83" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/0ae0faf9-64ae-4a05-b5a8-657a467a6907">